src/App.js

```diff
@@ -1,24 +1,22 @@
-import logo from './logo.svg';
+import './css-reset.css';
 import './App.css';
+import { BrowserRouter as Router, Route, Routes } from 'react-router-dom';
+import React from 'react';
+import Nav from './navigation/Nav';
+import Profile from './pages/Profile';
+import Missions from './pages/mission/Missions';
+import Rockets from './pages/Rockets';
 
 function App() {
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
+    <Router>
+      <Nav />
+      <Routes>
+        <Route path="/" element={<Rockets />} />
+        <Route path="/profile" element={<Profile />} />
+        <Route path="/mission" element={<Missions />} />
```

Comment:
✅ Proper routes and view components should be created (Rockets, Missions, My Profile, 👍🏻